pinot-core/src/test/java/com/linkedin/pinot/core/plan/CombinePlanNodeTest.java

```diff
@@ -36,7 +36,7 @@
   @Test
   public void testParallelExecution() {
     AtomicInteger count = new AtomicInteger(0);
-    int numPlans = 42;
+    int numPlans = 99;
```

Comment:
may also be useful to pick multiple numPlans values and run the test in a loop for a few different ones.